mesonbuild/mformat.py

```diff
@@ -979,7 +986,10 @@ def run(options: argparse.Namespace) -> int:
             src_file = src_file / build_filename
 
         try:
-            code = src_file.read_text(encoding='utf-8')
+            if from_stdin:
+                code = sys.stdin.read()
```

Comment:
I think the error message looks funny for the stdin case.

Reply:
Good point! :+1: